### PR TITLE
Base64 encoding

### PR DESCRIPTION
**Pull request by bowenwang1996**
Supports encoding from `Uint8Array` and decoding to `Uint8Array`.

### CODE REVIEW

File: near.ts [hunk 1]
```diff
@@ -1538,3 +1538,97 @@ declare function received_amount(): u64;
...
+    }
+
+    return x;
+  }
+}
+
+/**
+ * An instance of the Base64 class that is used for base64 encoding/decoding.
+ */
+export let base64: Base64 = new Base64();
```
Comment (vgrichina) [inline near.ts]: just declare it as a namespace similar to `near`? There is no point to have class instance for pure functions.

File: near.ts [hunk 2]
```diff
@@ -1538,3 +1538,97 @@ declare function received_amount(): u64;
...
+    let x = "",
+        imax = bytes.length - bytes.length % 3;
+
+    if (bytes.length == 0) {
+      return "";
+    }
+
+    for (i = 0; i < imax; i += 3) {
+      b10 = (bytes[i] as u32 << 16) | (bytes[i+1] as u32 << 8) | bytes[i+2] as u32;
+      x = x.concat(
```
Comment (vgrichina) [inline near.ts]: using `.concat` is a performance regression (results in `O(N^2)` complexity) compared to the code being ported (which used array and `.join`).

**Comment by vgrichina** [comment]
I think it makes sense to add some tests using https://github.com/jtenner/as-pect 
Doesn't have to block this PR though.

File: near.ts (hunk 1 above)
Comment (bowenwang1996) [reply near.ts]: Fixed

File: near.ts (hunk 2 above)
Comment (bowenwang1996) [reply near.ts]: Fixed

**Comment by bowenwang1996** [comment]
Made an issue for this #17

**Pull request merged**